InputTime/index.jsx

```diff
@@ -84,8 +86,8 @@ const InputTime = ({
       marginBottom: 0,
       marginLeft: 0,
       padding: 0,
-      '-webkit-appearance': 'none',
-      '-moz-appearance': 'none',
+      WebkitAppearance: 'none',
```

Comment:
Great point! Yeah, I was thinking whether or not to update this in a new branch. I'll definitely isolate these small pieces in new branches or separate commits. Thanks for the nudge there! 😅

Yeah, here's a screenshot of how it's rendering in buffer-web:
<img width="792" alt="screen shot 2017-06-01 at 9 34 13 pm" src="https://cloud.githubusercontent.com/assets/1685602/26699610/632301c2-4712-11e7-968a-e46535037eba.png">

 We likely won't be able to drop `noStyle` until perhaps buffer-web goes full-width. I'm keen to explore other options here to keep `InputTime` clean!